Habitica/src/triggers/TaskPointCalculation.trigger

```diff
@@ -1,12 +1,16 @@
 trigger TaskPointCalculation on Task__c (after insert, after update) {
+    String taskCompleted = 'Completed';
```

Comment:
If this is an option from a picklist, please use Final String. Do this for all these similar variables.